composer.json

```diff
@@ -11,8 +11,11 @@
         }
     ],
     "require": {
-        "php": ">=5.4.0",
-        "illuminate/support": "4.2.*|5.*"
+        "php": ">=5.4.0"
+    },
+    "require-dev": {
+        "phpunit/phpunit": "^7",
+        "orchestra/testbench": "^3.6"
```

Comment:
Don't know why it appears in this PR? I think there is a mix between two PR, maybe due to a merge.